apps/aehttp/src/aehttpc_doge.erl

```diff
@@ -193,67 +94,17 @@ getblock(NodeSpec, Seed, Hash, _Verbosity) ->
         {error, {E, R, S}}
     end.
 
-listunspent(NodeSpec) ->
-    try
-        Seed = <<>>,
-        Body = jsx:encode(request_body(<<"listunspent">>, [0, 9999999], seed_to_utf8(Seed))),
-        {ok, Res} = request(<<"/">>, Body, NodeSpec, 5000),
-        Unspent = result(Res),
-        {ok, Unspent}
-    catch E:R ->
-        {error, {E, R}}
-    end.
-
-createrawtransaction(NodeSpec, Inputs, Outputs) ->
-    try
-        Seed = <<>>,
-        Body = jsx:encode(request_body(<<"createrawtransaction">>, [Inputs, Outputs], seed_to_utf8(Seed))),
-        {ok, Res} = request(<<"/">>, Body, NodeSpec, 5000),
-        Tx = result(Res),
-        {ok, Tx}
-    catch E:R ->
-        {error, {E, R}}
-    end.
-
-%% Rely on the wallet embedded in the local bitcoind
-%% FIXME: Allow this to use a different host/port so users can use a separate offline bitcoind
-%% holding their commitments wallet.
--spec signrawtransaction(aehttpc:node_spec(), binary()) -> {ok, binary()} | {error, term()}.
-signrawtransaction(NodeSpec, RawTx) ->
-    try
-        Seed = <<>>,
-        Body = jsx:encode(request_body(<<"signrawtransaction">>, [RawTx], seed_to_utf8(Seed))),
-        {ok, Res} = request(<<"/">>, Body, NodeSpec, 5000),
-        SignedTx = result(Res),
-        Complete = maps:get(<<"complete">>, SignedTx),
-        true = Complete,
-        Hex = maps:get(<<"hex">>, SignedTx),
-        {ok, Hex}
-    catch E:R ->
-        {error, {E, R}}
-    end.
-
--spec sendrawtransaction(aehttpc:node_spec(), binary()) -> {ok, binary()} | {error, term()}.
-sendrawtransaction(NodeSpec, Hex) ->
-    try
-        Seed = <<>>,
-        Body = jsx:encode(request_body(<<"sendrawtransaction">>, [Hex], seed_to_utf8(Seed))),
-        {ok, Res} = request(<<"/">>, Body, NodeSpec, 5000),
-        {ok, result(Res)}
-    catch E:R ->
-        {error, {E, R}}
-    end.
 
--spec getrawtransaction(aehttpc:node_spec(), binary()) -> {ok, binary()} | {error, term()}.
-getrawtransaction(NodeSpec, TxHash) ->
-    try
-        Seed = <<>>,
-        Body = jsx:encode(request_body(<<"getrawtransaction">>, [TxHash, true], seed_to_utf8(Seed))),
-        {ok, Res} = request(<<"/">>, Body, NodeSpec, 5000),
-        {ok, result(Res)}
-    catch E:R ->
-        {error, {E, R}}
-    end.
+% -spec getrawtransaction(aehttpc:node_spec(), binary()) -> {ok, binary()} | {error, term()}.
```

Comment:
Possibly useful as well when dealing with dodge as parent chain?

Reply:
Again, I can reintroduce all the pruned code from these aehttpc modules (or whatever they are called) and export them. then these would make more sense. And not trigger build fails I suppose?